core/src/components/toast/toast.tsx

```diff
@@ -193,6 +196,41 @@ export class Toast implements ComponentInterface, OverlayInterface {
    */
   @Prop() htmlAttributes?: { [key: string]: any };
 
+  /**
+   * If set to 'vertical', the Toast can be dismissed with
+   * a swipe gesture. The swipe direction is determined by
+   * the value of the `position` property:
+   * `top`: The Toast can be swiped up to dismiss.
+   * `bottom`: The Toast can be swiped down to dismiss.
+   * `middle`: The Toast can be swiped up or down to dismiss.
+   */
+  @Prop() swipeGesture?: 'vertical';
```

Comment:
~ I'm ok with this as-is, but any reason for not specifying the entire type as part of this PR? You'll have to update this again to add `top`, `bottom` and `middle` which will create an additional diff to review vs. being reviewed as part of this.